tests/output.rs

```diff
@@ -714,3 +717,333 @@ fn test_overlapping_matches_vimgrep() {
         "#,
     );
 }
+
+#[test]
+fn test_after_context() {
+    assert_sorted_output(
+        "rust_project",
+        r#"
+            $ tree-sitter-grep -q '(function_item) @f' -l rust --after-context 2
+            src/stop.rs:1:fn stop_it() {}
+            src/helpers.rs:1:pub fn helper() {}
+            src/lib.rs:3:pub fn add(left: usize, right: usize) -> usize {
+            src/lib.rs:4:    left + right
+            src/lib.rs:5:}
+            src/lib.rs-6-
+            src/lib.rs-7-#[cfg(test)]
+            --
```

Comment:
I don't know what the exact "rules" are for when it throws in one of these `--` "context separator lines" but I feel pretty confident that we're getting the same behavior as `grep`/`ripgrep`